SnapMD.VirtualCare.ApiModels/Scheduling/AppointmentsRequest.cs

```diff
@@ -52,5 +52,15 @@ public class AppointmentsRequest : AvailabilityBlocksRequest
         /// The include patient co-userss.
         /// </value>
         public bool? IncludePatientCoUsers { get; set; }
+
+        /// <summary>
+        /// Records to skip.
+        /// </summary>
+        public int? Skip { get; set; }
```

Comment:
@lorddev Everything from `AppointmentsRequest` comes to Web API from query string including skip/take.